client/my-sites/stats/activity-log-banner/progress-banner.jsx

```diff
@@ -44,50 +44,67 @@ function ProgressBanner( {
 	restoreId,
 	downloadId,
 	action,
+	context,
 } ) {
-	return (
-		<ActivityLogBanner
-			status="info"
-			title={
-				'restore' === action
-					? translate( 'Currently restoring your site' )
-					: translate( 'Currently creating a downloadable backup of your site' )
+	let title = '';
+	let description = '';
+	let statusMessage = '';
+
+	const dateTime = applySiteOffset( moment.utc( ms( timestamp ) ) ).format( 'LLLL' );
+
+	switch ( action ) {
+		case 'restore':
+			if ( 'alternate' === context ) {
+				title = translate( 'Currently cloning your site' );
+				description = translate(
+					"We're in the process of cloning your site to %(dateTime)s. " +
+						"You'll be notified once it's complete.",
+					{ args: { dateTime } }
+				);
+				statusMessage =
+					'queued' === status
+						? translate( 'The cloning process will start in a moment.' )
+						: translate( "We're on it! Your site is being cloned." );
+			} else {
+				title = translate( 'Currently restoring your site' );
+				description = translate(
+					"We're in the process of restoring your site back to %(dateTime)s. " +
+						"You'll be notified once it's complete.",
+					{ args: { dateTime } }
+				);
+				statusMessage =
+					'queued' === status
+						? translate( 'Your restore will start in a moment.' )
+						: translate( "We're on it! Your site is being restored." );
 			}
-		>
-			{ 'restore' === action && (
-				<div>
+			break;
+
+		case 'backup':
+			title = translate( 'Currently creating a downloadable backup of your site' );
+			description = translate(
+				"We're in the process of creating a downloadable backup of your site at %(dateTime)s. " +
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'We're in the process of creating a downloadable backup of your site at %s. You'll be notified once it's complete.' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).